src/components/LocationPage/ChartBlock.tsx

```diff
@@ -25,44 +25,38 @@ function ChartBlock(props: {
   shareButtonProps: { [key: string]: any };
   metric: Metric;
   projections: Projections;
-  data?: any;
```

Comment:
I did some minor refactoring (elevated projections, metric, isMobile to local variables) but the only major change here is removing `data` and instead relying on `projections.hasMetric(metric)` to decide if there is data.